SECURITY.md

```diff
@@ -0,0 +1,3 @@
+# Security Policy
+
+If you discover any security related issues, please email baris@thundra.io .
```

Comment:
`support@runsidekick.com` mail is more suitable

Reply:
you are right. fixed it